utils/misc/misc.go

```diff
@@ -1247,16 +1250,18 @@ func WithBugsnagForWarehouse(fn func() error) func() error {
 func BugsnagNotify(ctx context.Context, team string) func() {
 	return func() {
 		if r := recover(); r != nil {
-			defer bugsnag.AutoNotify(ctx, bugsnag.SeverityError, bugsnag.MetaData{
-				"GoRoutines": {
-					"Number": runtime.NumGoroutine(),
-				},
-				"Team": {
-					"Name": team,
-				},
+			notifyOnce.Do(func() {
```

Comment:
missing defer

Reply:
since, we have moved `RecordAppError` inside, I don't think we need defer anymore.

Reply:
fine, as long as `RecordAppError` doesn't panic!